extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/WrapperExtensionContextBuilder.java

```diff
@@ -128,13 +128,19 @@ public static WrapperExtensionContext buildContext(
                 atomicConstraintMapper,
                 typeTransformerRegistry);
         var edcPropertyUtils = new EdcPropertyUtils();
-        var assetBuilder = new UiAssetMapper(edcPropertyUtils, jsonLd);
+        var assetBuilder = new UiAssetMapper(edcPropertyUtils);
```

Comment:
Perhaps uiAssetMapper is a better Naming, we already have uiAssetBuilder != uiAssetMapper